dockerfiles/hack/run-integration.py

```diff
@@ -99,4 +96,7 @@ def run_cmd():
         if return_code == State.ERROR:
             continue
 
+        if RUN_ONCE:
+            sys.exit(return_code)
+
```

Comment:
Why you want to skip the `if RUN_ONCE` check in case of error? Then it will not run only once...

Reply:
i guess that makes sense. another integration that is running once is `github-users` (which is sending emails). we don't want our users to be spammed in case of an error.

moved it to before the error check.

Reply:
the important integration to think of here is `aws-ecr-image-pull-secrets`. but in any case, we can leave it up for the CronJob to retry.